setup.cfg

```diff
@@ -116,6 +116,12 @@ windows_platform =
 osx_platform =
     %(macos_platform)s
 
+mypy =
+    mypy==0.812
+    mypy-zope==0.2.13
+    %(dev)s
+    %(all_non_platform)s
```

Comment:
Ew `%`-formatting. `setup.cfg` doesn't allow f-strings?

Reply:
This is an f-string–like interpolation, really, it's not `%`-formatting: https://docs.python.org/3/library/configparser.html#interpolation-of-values